saleor/graphql/schema.graphql

```diff
@@ -2725,9 +2704,6 @@ type Mutation {
   draftOrderDelete(id: ID!): DraftOrderDelete
   draftOrderBulkDelete(ids: [ID]!): DraftOrderBulkDelete
   draftOrderLinesBulkDelete(ids: [ID]!): DraftOrderLinesBulkDelete
-  draftOrderLinesCreate(id: ID!, input: [OrderLineCreateInput]!): DraftOrderLinesCreate
```

Comment:
The info about removing these mutations should be placed in Changelog in breakingChanes section. Someone could have an app which uses this.